index.html

```diff
@@ -1,348 +1,331 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css" />
-    <title>Tommymarvel Portfolio</title>
-  </head>
 
-  <body>
-    <div class="firstpage">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="styles.css" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+
+  <title>Tommymarvel Portfolio</title>
+</head>
+
+<body>
+  <div class="firstpage">
     <header class="white">
       <a class="logo" href="#">My logo</a>
-      <nav class="nav-bar">
-        <img id="hambuger" src="images/Union.png" alt="hamburger" />
+      <nav class="nav-bars">
+        <div id="hamburger">
+          <i class="fa fa-bars"></i>
+        </div>
+        <div class="mobile-menu">
+          <a href="#" class="close-icon" id="closemenu"><img src="/images/Icon.png" alt="X icon">
+            ></a>
+          <ul class="mobile-nav">
+            <li class="nav-item"><a href="#work-section-link" class="nav-link">Portfolio</a></li>
+            <li class="nav-item"><a href="#about-section-link" class="nav-link">About</a></li>
+            <li class="nav-item"><a href="#contact-form-link" class="nav-link">Contact</a></li>
+          </ul>
+        </div>
         <ul class="nav-tags">
```

Comment:
- [ ] Kindly make sure your **Mobile Menu** design sticks to the one from your chosen Figma template as required for the project and then your project will be complete(e.g.: background color, blur effect on the background, font family, opacity,...). Check everything to match your design to the expected one.

| Figma Design | Your Design |
| ------------------- | ------------------- |
| ![mmf1](https://user-images.githubusercontent.com/83042742/179861251-29fe7a55-1ce0-458d-96d9-0081305ae2f6.png) | ![Screenshot from 2022-07-20 00-50-41](https://user-images.githubusercontent.com/83042742/179861507-998072de-2d48-441f-9981-a38455adb4b2.png) |